TARS/src/mcu_main/sensors/GasSensor.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <ChRt.h>
+#include <Adafruit_BME680.h>
+
+#include "mcu_main/error.h"
+
+class GasSensor {
+public:
+    GasSensor();
+
+    ErrorCode __attribute__((warn_unused_result)) init();
+    void refresh();
+
+    float readTemperature();
+
+private:
+    Adafruit_BME680 bme;
```

Comment:
Turning off ENABLE_GAS_SENSOR should allow us to remove the lib_dep and still compile. Currently it doesn't do that, as evidenced by this line not being wrapped in guards (also the include above, and code inside of GasSensor.cpp).

Doing this will make it easier to adapt code when we switch what peripherals are on TARS.